api/handlers/grouphandler.py

```diff
@@ -22,6 +22,7 @@ def get(self, _id):
             self._filter_permissions([result], self.uid)
         if self.is_true('join_avatars'):
             ContainerHandler.join_user_info([result])
+        util.add_container_type(self.request, result)
```

Comment:
It will only show up if you set the `fw_container_type` key in the environment, so currently it's only for forwarded requests. I wouldn't have any complaints about that key showing up everywhere, so I'm open to that conversation; if that were the case I'd advocate moving the logic into `ContainerStorage`.